qute.ls/com.redhat.qute.ls/src/test/java/com/redhat/qute/services/diagnostics/QuteDiagnosticsSeverityTest.java

```diff
@@ -0,0 +1,65 @@
+/*******************************************************************************
+* Copyright (c) 2022 Red Hat Inc. and others.
+* All rights reserved. This program and the accompanying materials
+* which accompanies this distribution, and is available at
+* http://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Contributors:
+*     Red Hat Inc. - initial API and implementation
+*******************************************************************************/
+package com.redhat.qute.services.diagnostics;
+
+import static com.redhat.qute.QuteAssert.d;
+import static com.redhat.qute.QuteAssert.testDiagnosticsFor;
+
+import org.eclipse.lsp4j.Diagnostic;
+import org.eclipse.lsp4j.DiagnosticSeverity;
+import org.junit.jupiter.api.Test;
+
+import com.redhat.qute.settings.QuteValidationSettings;
+import com.redhat.qute.settings.QuteValidationTypeSettings;
+
+/**
+ * Test with diagnostic severity setting
+ *
+ */
+public class QuteDiagnosticsSeverityTest {
+
+	@Test
+	public void undefinedVariableSeverityDefault() throws Exception {
+		String template = "{foo}";
+		QuteValidationSettings validationSettings = new QuteValidationSettings();
+		Diagnostic d = d(0, 1, 0, 4, QuteErrorCode.UndefinedObject, //
+				"`foo` cannot be resolved to an object.", DiagnosticSeverity.Warning);
+		d.setData(DiagnosticDataFactory.createUndefinedVariableData("foo", false));
+		testDiagnosticsFor(template, validationSettings, d);
+	}
+
+	@Test
+	public void undefinedVariableSeverityWarning() throws Exception {
+		String template = "{foo}";
+		QuteValidationSettings validationSettings = new QuteValidationSettings();
+		QuteValidationTypeSettings undefinedObject = new QuteValidationTypeSettings();
+		undefinedObject.setSeverity(DiagnosticSeverity.Warning.name());
+		validationSettings.setUndefinedObject(undefinedObject);
+		Diagnostic d = d(0, 1, 0, 4, QuteErrorCode.UndefinedObject, //
+				"`foo` cannot be resolved to an object.", DiagnosticSeverity.Warning);
+		d.setData(DiagnosticDataFactory.createUndefinedVariableData("foo", false));
+		testDiagnosticsFor(template, validationSettings, d);
+	}
+
+	@Test
+	public void undefinedVariableSeverityError() throws Exception {
+		String template = "{foo}";
+		QuteValidationSettings validationSettings = new QuteValidationSettings();
+		QuteValidationTypeSettings undefinedObject = new QuteValidationTypeSettings();
+		undefinedObject.setSeverity(DiagnosticSeverity.Error.name());
```

Comment:
Do https://github.com/redhat-developer/quarkus-ls/pull/587/files#r824092220 towrite

```
import static ...QuteValidationSettings.Sverity.

undefinedObject.setSeverity(Severity.error.name());

```
do thzt for the other tests.